internal/wrappers/mock/tenant-mock.go

```diff
@@ -15,5 +15,9 @@ func (t TenantConfigurationMockWrapper) GetTenantConfiguration() (
 			Key:   "scan.config.plugins.ideScans",
 			Value: "true",
 		},
+		{
+			Key:   "scan.config.plugins.aiGuidedRemediation",
```

Comment:
I see the mock, but please add a test too (unit), and make sure that the integration test cover this call too (happy case)